environment/main.cpp

```diff
@@ -130,7 +130,7 @@ int main(int argc, char ** argv) {
         std::cout << stats;
     }
     else {
-        for(unsigned int a = 0; a < stats.player_statistics.size(); a++) std::cout << "Player #" << stats.player_statistics[a].tag << ", " << my_game->getName(stats.player_statistics[a].tag) << ", came in rank #" << stats.player_statistics[a].rank << "!\n";
+        for(unsigned int a = 0; a < stats.player_statistics.size(); a++) std::cout << "Player #" << stats.player_statistics[a].tag << ", " << my_game->getName(stats.player_statistics[a].tag) << ", came in rank #" << stats.player_statistics[a].rank << ", and was last alive on frame #" << stats.player_statistics[a].last_frame_alive << "!\n";
```

Comment:
That's known as an Oxford comma (or sometimes Serial comma) and is optional depending in the style guide you are using. American style guides more commonly require it and British ones less commonly (The Oxford Style Manual as the comma name suggests being one of the minority British guides).

In other words it is correct with or without it, unless you need to follow a specific style that requires one way or the other.